de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/ddl/functions/customerItemStatistics.sql

```diff
@@ -118,15 +112,15 @@ WITH t AS (
 SELECT t.BPValue,
        t.productValue,
        t.Name,
-       t.qtyInvoiced,
+       round(t.qtyInvoiced, 2)               AS qtyInvoiced,
```

Comment:
hmmm..... why?

Reply:
Got rid of the rounding and added a cast to numeric so we are sure there won't be any trailing zeros.